src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,205 @@
+
+```css
+.brighten-saturate-and-blur-behind-me {
+  backdrop-filter: brightness(150%) saturate(150%) blur(1rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-ambient_blur.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Try this example for yourself in <a href="https://codepen.io/pepf/pen/GqZkdj" target="_blank">CodePen</a>.
+  </figcaption>
+</figure>
+
+## Creative examples
+
+Here are some interesting capabilities unlocked with `backdrop-filter`. I find these examples are best for teaching what could beß.
+
+### Overlays
+
+This example shows how to blur a semi-transparent background to make text readable while stylistically blending with a page's background.
+
+```css
+.modal {
+  backdrop-filter: blur(10px);
+  background-color: rgba(255, 255, 255, 0.5);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-modal.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Try this <a href="https://mfreed7.github.io/backdrop-filter-feature/examples/scrollable.html" target="_blank">example</a> for yourself.
+  </figcaption>
+</figure>
+
+### Text contrast on dynamic backgrounds
+
+I said earlier that `backdrop-filter` allows performant effects that would be difficult or impossible on the web. The following example shows how to change a background in reponse to an animation. This is done to maintain high contrast between text and its background in spite of what's going on behind the text.
+
+This animation starts with the default background color `darkslategray` and uses `backdrop-filter` to invert the colors after the transformation.
+
+```css
+.container::before {
+  z-index: 1;
+  background-color: darkslategray;
+  filter: invert(1);
+}
+
+.container::after {
+	-webkit-backdrop-filter: invert(1);
+  backdrop-filter: invert(1);
+  z-index: 3;
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-invert_color.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Try this example from <a href="https://www.chenhuijing.com/#%F0%9F%91%9F">Chen Hui Jing</a> in <a href="https://tympanus.net/codrops-playground/huijing/Qqpwg5Iy/editor" target="_blank">Codrops</a>.
+  </figcaption>
+</figure>
+
+## Conclusion
+More than 560 of you have upvoted the [Chromium bug](https://crbug.com/497522) over the past few years, clearly marking this as a long awaited CSS feature. The wait is over. Chrome's excited to release `backdrop-filter` in version 76.
```

Comment:
```suggestion
This post shows only a few of the exciting effects that `backdrop-filter` makes possible. It really is a game changer for visual styles on the web. And with its imminent arrival in Chrome 76—and [more browser support likely on the way](https://wiki.mozilla.org/CSS#2019_Medium_Priorities)—the web is another step closer to truly native-like UI presentation. 
```
Given (a) we're trying to be browser agnostic, and (b) Firefox doesn't yet support backdrop-filter, I tried to reframe this a bit. Not totally happy with it to be honest, so please feel free to make it better!

Reply:
Noting the upvotes isn't as Chrome centric as it seems. There's a subtext to this comment you need to be aware of. 500 is an unusual number of stars for a Chrome bug. Responding to that unusual number is a demonstration that we support the platform for its own sake, not just for our own profit.

I've added some of your language.